Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py

```diff
@@ -183,11 +182,15 @@ def get_user(self, user, properties):
         except Exception as e:
             raise e
 
-    def list_users(self, properties, page_url):
+    def list_users(self, properties, page_url, filters):
         if page_url:
             response = self.ms_client.http_request(method='GET', url_suffix='users', full_url=page_url)
```

Comment:
add `params={'$filter': filters}` here

Reply:
For both of these comments on the list_users command I think you're suggested changes break the functionality I am trying to add. My understanding is that with the page_url argument is present when it is input into the API all the API does is get the next page of the results from a previous API call. If we removed the filters elif and added the filters to the page_url if then we essentially remove the functionality of the filters addition from my understanding.

Reply:
ok so just add the filters in the second request:
```
        if page_url:
            response = self.ms_client.http_request(method='GET', url_suffix='users', full_url=page_url)
        else:
            response = self.ms_client.http_request(method='GET', url_suffix='users', params={'$select': properties,'$filter': filters})
```

Reply:
Ok I removed the elif filters the else: still needs the headers I added and the $count parameter for the filter parameter to work, if no filters are entered the command still works as it used to before the changes.